server/server.js

```diff
@@ -0,0 +1,23 @@
+var express = require('express');
+var path = require('path');
+var bodyParser = require('body-parser');
+
+var app = express();
+
+app.use(bodyParser.json());
+
+app.get('/card', (req, res) => {
+  res.sendFile(path.join(__dirname, '..', 'clients', 'sender', 'index.html'));
```

Comment:
The `path.join()` would look better just like `path.join(__dirname, '../clients/sender/index.html');`.

The `path.join()` function doesn't seem much different than doing `__dirname + '../clients/sender/index.html'`, but you would be surprised how often a missing or extra '/' can totally break your code. Path.join is more of just a safeguard for this.